### PR TITLE
PMM-5842 Add nice error for create-user flag.

### PR DESCRIPTION
**Pull request by JiriCtvrtka**

### CODE REVIEW

File: commands/management/add_mysql.go [hunk 1]
```diff
@@ -268,5 +274,6 @@ func init() {
 	AddMySQLC.Flag("skip-connection-check", "Skip connection check").BoolVar(&AddMySQL.SkipConnectionCheck)
 	AddMySQLC.Flag("tls", "Use TLS to connect to the database").BoolVar(&AddMySQL.TLS)
 	AddMySQLC.Flag("tls-skip-verify", "Skip TLS certificates validation").BoolVar(&AddMySQL.TLSSkipVerify)
+	AddMySQLC.Flag("create-user", "Create user root").Hidden().BoolVar(&AddMySQL.CreateUser)
```
Comment (BupycHuk) [inline commands/management/add_mysql.go]: flag should be hidden for now

Comment (JiriCtvrtka) [reply commands/management/add_mysql.go]: It is hidden in help, so I am not sure how you mean it?

Comment (BupycHuk) [reply commands/management/add_mysql.go]: oh, sorry missed that :+1:

Comment (JiriCtvrtka) [reply commands/management/add_mysql.go]: No problem.

File: commands/management/add_mysql_test.go [hunk 2]
```diff
@@ -133,4 +134,48 @@ Table statistics collection disabled (always).
...
+		_, err := cmd.Run()
+
+		if assert.Error(t, err) {
+			expected := errors.New("Unrecognized option. To create a user, see'https://www.percona.com/doc/percona-monitoring-and-management/2.x/concepts/services-mysql.html#pmm-conf-mysql-user-account-creating'")
+			assert.Equal(t, expected, err)
+		}
+	})
+}
+
+func TestGetAddress(t *testing.T) {
```
Comment (BupycHuk) [inline commands/management/add_mysql_test.go]: Do we really need these tests?

**Review by BupycHuk — APPROVED** [review]

File: commands/management/add_mysql.go (hunk 1 above)
Comment (askomorokhov) [inline commands/management/add_mysql.go]: This is minor but I guess it's better to be
```suggestion
	AddMySQLC.Flag("create-user", "Create pmm user").Hidden().BoolVar(&AddMySQL.CreateUser)
```
according to https://www.percona.com/doc/percona-monitoring-and-management/conf-mysql.html#pmm-conf-mysql-user-account-creating

File: commands/management/add_mysql.go [hunk 3]
```diff
@@ -163,6 +164,11 @@ func (cmd *addMySQLCommand) Run() (commands.Result, error) {
 		return nil, err
 	}
 
+	if cmd.CreateUser {
+		return nil, fmt.Errorf("Unrecognized option. To create a user, see" +
```
Comment (askomorokhov) [inline commands/management/add_mysql.go]: I think it will look better with one space between `... see'https...`

**Review by askomorokhov — APPROVED** [review]
Looks good to me with comments.

File: commands/management/add_mysql.go (hunk 3 above)
Comment (JiriCtvrtka) [reply commands/management/add_mysql.go]: Sure. I miss that.

File: commands/management/add_mysql_test.go (hunk 2 above)
Comment (JiriCtvrtka) [reply commands/management/add_mysql_test.go]: It actually not increase patch coverage, so no. Removed.

**Pull request merged**